sdk/python/feast/infra/online_stores/contrib/cassandra_online_store/cassandra_online_store.py

```diff
@@ -566,8 +570,18 @@ def _create_table(self, config: RepoConfig, project: str, table: FeatureView):
         session: Session = self._get_session(config)
         keyspace: str = self._keyspace
         fqtable = CassandraOnlineStore._fq_table_name(keyspace, project, table)
-        create_cql = self._get_cql_statement(config, "create", fqtable)
-        logger.info(f"Creating table {fqtable}.")
+
+        ttl = (
+            table.online_store_ttl
+            if table.online_store_ttl is not None
+            else config.online_store.ttl
```

Comment:
Are we using the ttl set in feature_store.yaml here if there is no ttl set at featureview/table level?

Reply:
Yep, that's what this is doing